modules/Hoot/ui/conflicts/info.js

```diff
@@ -26,19 +27,35 @@ export default class ConflictMetadata {
             /source:ingest:datetime/,
             /uuid/
         ];
+
+        Hoot.context.history().on('change.reviewtagtable',
+            _debounce( () => this.buildTagTable.bind(this)(), 300)
+        );
     }
 
     /**
      * Create tag table for revieawble items
      */
     buildTagTable() {
-        let colData    = this.data.currentFeatures,
-            tags1      = this.filterTags( colData[ 0 ] ? colData[ 0 ].tags : {} ),
+        let currentRelation;
+        try {
+            currentRelation = this.instance.graphSync.getCurrentRelation();
+        } catch (e) {
+            //current relation not found which is ok
+        }
+        //if triggered by history change and not in review mode return
+        if (!currentRelation) return;
+
+        let features = this.data.currentFeatures,
+            feature = Hoot.context.hasEntity( features[0].id ),
+            againstFeature = Hoot.context.hasEntity( features[1].id );
+
+        let colData = [feature, againstFeature];
```

Comment:
@jackgrossman18 this seems to me a simpler way to handle if a feature has been merged and 1) had tags updated or 2) been deleted.  By grabbing the features from the graph instead of the stale `currentFeatures` the tag table will now update correctly anytime review merges happen.